### ISSUE TITLE
Pyramid crashes when try to initialize cliquet

### ISSUE THREAD

**Issue by rodo** [mozilla-services/cliquet#489]
I followed step by step the Quickstart (as I'm a noob in Pyramid and Cliquet) http://cliquet.readthedocs.org/en/latest/quickstart.html and I block on something look like a bug.

When I add `cliquet.initialize(config, __version__)` as describe in the doc Pyramid crashes with the following trace

* Python 2.7.8

```
-------------------- Restarting --------------------
/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/cliquet/initialization.py:443: UserWarning: No value specified for `project_name`
  warnings.warn('No value specified for `project_name`')
Traceback (most recent call last):
  File "/home/rodo/dev/public/clidemo/venv/bin/pserve", line 11, in <module>
    sys.exit(main())
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 58, in main
    return command.run()
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 328, in run
    global_conf=vars)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/pyramid/scripts/pserve.py", line 363, in loadapp
    return loadapp(app_spec, name=name, relative_to=relative_to, **kw)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 247, in loadapp
    return loadobj(APP, uri, name=name, **kw)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 272, in loadobj
    return context.create()
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 710, in create
    return self.object_type.invoke(self)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/loadwsgi.py", line 146, in invoke
    return fix_call(context.object, context.global_conf, **context.local_conf)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/util.py", line 58, in fix_call
    reraise(*exc_info)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/compat.py", line 23, in reraise
    exec('raise t, e, tb', dict(t=t, e=e, tb=tb))
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/paste/deploy/util.py", line 55, in fix_call
    val = callable(*args, **kw)
  File "/home/rodo/dev/public/clidemo/foopi/foopi/__init__.py", line 16, in main
    cliquet.initialize(config, __version__)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/cliquet/initialization.py", line 450, in initialize
    load_default_settings(config, cliquet_defaults)
  File "/home/rodo/dev/public/clidemo/venv/local/lib/python2.7/site-packages/cliquet/initialization.py", line 398, in load_default_settings
    distinct_values = set([settings[d] for d in defined])
TypeError: unhashable type: 'list'
```
The only lines I added to `development.ini` 

```
cliquet.cache_backend = cliquet.cache.memory                                        
cliquet.storage_backend = cliquet.storage.memory                                    
cliquet.permission_backend = cliquet.permission.memory 
```

Can you help me ?

**Comment by Natim** [comment]
Yes this is a bug I fixed few days ago. Here: https://github.com/mozilla-services/cliquet/commit/92b3f8c9730b0889e18f1e8f34ad46211a7b252a

I merged it this morning. If you are using Cliquet master it should work. I will plan a bugfix release Thursday.

**Comment by rodo** [comment]
Thank you @Natim it works now.

**Comment by leplatrem** [comment]
Fixed in 2.8.2
